build/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
 
 RUN mkdir -p /opt/jsonnet && chown nobody /opt/jsonnet
 
+RUN microdnf update
```

Comment:
This balloons the size of the image by duplicating content from earlier layers.

Upgrading packages in a container image is generally considered a poor practice. It would be better to use a base image that doesn't have the older packages in the first place.